torch/_inductor/config.py

```diff
@@ -175,6 +175,11 @@ def prologue_fusion_enabled() -> bool:
 # incompatible with disable_cpp_codegen
 cpp_wrapper: bool = os.environ.get("TORCHINDUCTOR_CPP_WRAPPER", "0") == "1"
 
+# compile entry and kernel separately
```

Comment:
Expand this description to explain why this is needed.

Reply:
Thanks for your comments. Added more descriptions